### PR TITLE
RUN-5188/ abstract out webcontents apis in the adapter

### PR DESCRIPTION
**Pull request by pbaize**
This can be taken independently of core changes, takes the webcontents related apis that will be shared with BrowserView and creates a new base class with them. Docs are still on Window

### CODE REVIEW

File: src/api/webcontents/webcontents.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import { EmitterBase } from '../base';
+import { BaseEventMap } from '../events/base';
+import { Identity } from '../../identity';
+import Transport from '../../transport/transport';
+
+export class WebContents<T extends BaseEventMap> extends EmitterBase<T> {
+    constructor(wire: Transport, identity: Identity, public entityType: string,) {
```
Comment (licui3936) [inline src/api/webcontents/webcontents.ts]: extra comma

File: src/api/webcontents/webcontents.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    /**
+     * Stops any current navigation the window is performing.
+     * @return {Promise.<void>}
+     * @tutorial Window.stopNavigation
+     */
+    public stopNavigation(): Promise<void> {
+        return this.wire.sendAction('stop-window-navigation', Object.assign({}, this.identity)).then(() => undefined);
+    }
+}
```
Comment (licui3936) [inline src/api/webcontents/webcontents.ts]: All the tutorials are still using Window, not WebContent.  Is it confusing?

Comment (pbaize) [reply src/api/webcontents/webcontents.ts]: The WebContent class will not be exposed, so right now it's actually undocumented. I had left these in to keep some documentation with the code, but maybe it's better if I remove it?

Comment (licui3936) [reply src/api/webcontents/webcontents.ts]: It will be exposed if you leave the jsdoc here.

**Comment by pbaize** [comment]
> 👍 can we confirm running the node tests with this looks ok?

They do

**Pull request merged**